### DISCUSSION TITLE
Can anyone fell, the app logout automatically

### DISCUSSION THREAD

**Discussion by AlanBinu007** [AlanBinu007/Realtime-Chat-Application-ReactJs#14] [Q&A] [answered]
Can anyone fell, the app logout automatically

**Comment by DanMathew7** [comment]
What happened?

**Reply by AlanBinu007** [reply]
One of my follower says that "It will automatically logout while using it" and i didn't fell anything like that

**Reply by DanMathew7** [reply, marked as answer]
No, i used it nothing like that

**Reply by AlanBinu007** [reply]
Thanks
